src/pivot-table/components/cells/ColumnAdjusterWrapper.tsx

```diff
@@ -35,7 +35,9 @@ const ColumnAdjusterWrapper = ({
 
   const updateWidth = (adjustedWidth: number) => {
     forceRerender({});
-    overrideLeftGridWidth?.(adjustedWidth, cellInfo.dimensionInfoIndex);
+    if (overrideLeftGridWidth && cellInfo.colIdx) {
```

Comment:
good catch. it is optional so need to sanity check it. I could cast it since if overrideLeftGridWidth exists, so does colIdx, but I think it is cleaner with an if statement